src/desktop/dialogs/joindialog.cpp

```diff
@@ -427,9 +415,9 @@ void JoinDialog::addListServer()
 	QString urlStr = m_ui->address->currentText();
 	QUrl url;
 	if(!urlStr.contains('/')) {
-		url = QUrl { "http://" + urlStr };
+		url = QUrl { "https://" + urlStr };
 	} else {
-		url = QUrl { "http://" + QUrl{urlStr}.host() };
+		url = QUrl { "https://" + QUrl{urlStr}.host() };
```

Comment:
I have updated this so it should only default to HTTPS if a scheme is not explicitly specified, instead of the old code which had been always forcing to insecure HTTP and which I had only just updated to always force to HTTPS instead.